cropimageview/build.gradle

```diff
@@ -1,14 +1,22 @@
 apply plugin: 'com.android.library'
-apply plugin: 'com.github.dcendents.android-maven'
-apply plugin: 'com.jfrog.bintray'
+apply plugin: 'com.novoda.bintray-release'
```

Comment:
It enables you uploading the lib to bintray + maven central (aar dependencies in our project)